enforcer/src/enforcer/enforce_task.h

```diff
@@ -35,10 +35,10 @@
 #include "scheduler/task.h"
 #include "db/policy.h"
 
-task_t *enforce_task(engine_type *engine, char const *owner);
+task_type *enforce_task(engine_type *engine, char const *owner);
 
 time_t enforce_task_perform(char const *owner, void *context,
-    db_connection_t *dbconn);
+    void *dbconn);
```

Comment:
Function signature differs from c file:
enforce_task_perform(char const *owner, void *userdata, void *context)
(also not sure at this point why dbconn is passed as void ptr.